packages/patternfly-4/react-core/src/components/Progress/Progress.tsx

```diff
@@ -1,60 +1,56 @@
-import React, { Component } from 'react';
+import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Progress/progress';
 import { css, getModifier } from '@patternfly/react-styles';
-import PropTypes from 'prop-types';
-import ProgressContainer, { ProgressMeasureLocation, ProgressVariant } from './ProgressContainer';
+import { ProgressContainer, ProgressMeasureLocation, ProgressVariant } from './ProgressContainer';
 import { getUniqueId } from '../../helpers/util';
+import { Omit } from '../../helpers/typeUtils';
 
-export const ProgressSize = {
-  sm: 'sm',
-  md: 'md',
-  lg: 'lg'
-};
+export enum ProgressSize {
+  sm = 'sm',
+  md = 'md',
+  lg = 'lg',
+}
 
-const propTypes = {
+export interface ProgressProps extends Omit<React.HTMLProps<HTMLDivElement>, 'size' | 'label'> {
   /** Classname for progress component. */
-  className: PropTypes.string,
+  className?: string;
   /** Size variant of progress. */
-  size: PropTypes.oneOf(Object.values(ProgressSize)),
+  size?: 'sm' | 'md' | 'lg';
   /** Where the measure percent will be located. */
-  measureLocation: PropTypes.oneOf(Object.values(ProgressMeasureLocation)),
+  measureLocation?: 'outside' | 'inside' | 'top' | 'none';
   /** Status variant of progress. */
-  variant: PropTypes.oneOf(Object.values(ProgressVariant)),
+  variant?: 'danger' | 'success' | 'info';
   /** Title above progress. */
-  title: PropTypes.string,
-  /** Text description of current progress value to display instead of
-   * percentage. */
-  label: PropTypes.node,
+  title?: string;
+  /** Text description of current progress value to display instead of percentage. */
+  label?: React.ReactNode;
   /** Actual value of progress. */
-  value: PropTypes.number,
+  value: number;
```

Comment:
In the previous TS this was optional

Reply:
Yeah, let's make it optional and just keep the sensible default of 0.